bravado/swagger_type.py

```diff
@@ -133,11 +151,18 @@ def is_array(type_):
     return type_.startswith(ARRAY + COLON)
 
 
+def is_object(type_):
+    """Checks whether the swagger type is object
+    :rtype: boolean
+    """
+    return type_ == 'object'
+
+
 def is_complex(type_):
     """checks whether the swagger type is neither primitive nor array
     :rtype: boolean
     """
-    return not is_primitive(type_) and not is_array(type_)
+    return (not is_primitive(type_) and not is_array(type_)) or is_object(type_)
```

Comment:
May be, it could be complex object, but how exactly is object and complex object different? I see two different create methods above.